add_npm_package.sh

```diff
@@ -72,13 +72,25 @@ if [[ -z $STRATEGY ]] ; then
 fi
 
 echo -n "Making directory..."
+UPDATE=false
+if [ -d "$PACKAGE_NAME" ]; then
```

Comment:
One thing I've run into a few times is that if the script fails the directory is created but there are no files in it. I'd prefer to check if the spec file actually exists. Especially since you're using it with sed later.